tests/utils_test.py

```diff
@@ -135,7 +140,10 @@ def check_rdf_scan(expected_file, result_file, regen=False):
     else:
         with io.open(expected_file, 'r', encoding='utf-8') as i:
             expected = sort_nested(json.load(i))
-    assert expected == result
+
+    expected_json = json.dumps(expected, sort_keys=True, indent=2)
+    result_json = json.dumps(result, sort_keys=True, indent=2)
+    assert result_json == expected_json
```

Comment:
Could this json conversion be avoided if the `sort_nested` function would sort dictionary entries by key (since we are dealing with `OrderedDict`s)?

Reply:
it is far easier to compare the json output side-by-side to see where the problems lie.